tetris.js

```diff
@@ -414,31 +414,34 @@ module.exports = {
 		return result.sort(this.orderCoordinates);
 	},
 	incrementPhase: function (typeOfTetromino, currentRotationPhaseOfTetromino) {
+		let arrayPhases = [];
 		if(typeOfTetromino === "I" || typeOfTetromino === "S" || typeOfTetromino === "Z"){
-			if(currentRotationPhaseOfTetromino === "vertical"){
-				return "horizontal"
-			}
-			if(currentRotationPhaseOfTetromino === "horizontal"){
-				return "vertical"
+			arrayPhases = ["vertical","horizontal"];
+			switch(currentRotationPhaseOfTetromino){
```

Comment:
Algorithm should be the same for all types, only arrayPhases are different.
Keep assignment to arrayPhases in ifs, and do algorithm after all ifs executed